server.go

```diff
@@ -16,6 +17,7 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
+// Route is ...
 type Route struct {
```

Comment:
An uppercase type is exported. go-lint demands, that it has a docstring. I was not able to find a good one :)